frontend/packages/dev-console/integration-tests/features/addFlow/quick-search-add.feature

```diff
@@ -0,0 +1,57 @@
+@add-flow @odc-6303
+Feature: Provide quick search in Add page
+              As a user, I should be able to have a quick way to search for items to add to my application/project in the Add page
+
+        Background:
+            Given user is at Add page
+              And user has created or selected namespace "aut-add"
+
+
+        @smoke @to-do
+        Scenario: Add to project button in Add page: AQS-01-TC01
```

Comment:
These scenarios are related to add flow, so can we have "A-" instead of "AQS-"
Same for remaining scenarios